src/NodeAFK.js

```diff
@@ -0,0 +1,55 @@
+const Listener = require('./Listener');
+
+class NodeAFK {
+  constructor() {
+    this.listeners = {};
+    this.nextListenerId = 1;
+  }
+
+  getAllListeners() {
+    return this.listeners;
+  }
+
+  // Add a listener to check when the user is away
+  // Interval - {Number} of Seconds
+  // callback - {Function}
+  // Returns a {Number} - The id of the listener that was created
+  addListener(interval, callback) {
+    const listenerId = this.nextListenerId;
+    this.nextListenerId += 1;
+
+    const afkListener = new Listener(listenerId, interval, callback);
+    afkListener.checkIsAway();
+
+    this.addListenerToList(listenerId, afkListener);
+
+    return listenerId;
+  }
+
+  addListenerToList(id, listener) {
```

Comment:
I don't think there is a need for this method. Its only used once and is doing something very trivial. The contents of this method could just be inlined.